FoodHaus_App/project-swift/Terms.txt

```diff
@@ -0,0 +1,24 @@
+Terms of Service
+Last Updated on November 27, 2012. These terms and conditions are effective immediately for those registering accounts after that date and will become effective December 27, 2012 for those with pre-existing accounts. To review the previous terms, please click here.
+
+Last Updated on November 27, 2012. These terms and conditions are effective immediately for those registering accounts after that date and will become effective December 27, 2012 for those with pre-existing accounts. To review the previous terms, please click here.
```

Comment:
Why the same paragraph repeats so many times? Also, I think it's better to delete 

> To review the previous terms, please click here.

@bojms45